test/functional/mempool_package_limits.py

```diff
@@ -26,7 +26,7 @@ def run_test(self):
         self.wallet = MiniWallet(self.nodes[0])
         # Add enough mature utxos to the wallet so that all txs spend confirmed coins.
         self.generate(self.wallet, 35)
-        self.generate(self.nodes[0], COINBASE_MATURITY)
+        self.generate(self.wallet, COINBASE_MATURITY)
```

Comment:
fell-free-to-ignore tiny-nit: could just do it in one call
```suggestion
        self.generate(self.wallet, 35 + COINBASE_MATURITY)
```